java/pom.xml

```diff
@@ -221,6 +217,26 @@
                     </dependency>
                 </dependencies>
             </plugin>
```

Comment:
Do we need this at all?

Reply:
> Do we need this at all?

Thinking again, probably not - the minimum version required (`3.0.1`) was [released in 2010](https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.0.1/), so it's unlikely anyone would be using an older version, and this [requirement was added in 2015](https://github.com/hazelcast/hazelcast-simulator/commit/7310feeffd1103450e8fdb87c9ddbb9bc77f7ccc).

Reply:
Just get rid of it.

Reply:
f990f40d3d251102baab8636ba781777ec12637b